scripts/time_to_peak.R

```diff
@@ -0,0 +1,37 @@
+#time to peak delF response for 1 pulse
+#open PlotGCaMP_multi csv file
+time_to_peak <- function(genotype, #mandatory genotype, should be text
+                                 pulse_length = 60, # time of pulse in seconds 
```

Comment:
'pulse' sounds a little too similar to stimulus imo, maybe change to pulse-loop or loop-length?

Reply:
done